iFixit/react-commerce
# Product Page Cache API Optimization

Closes #1523

- Replaces `fetch(); sleep();` with `request().end()` so we can resume the serverless function as soon as the request completes.
- Makes revalidation requests against the vercel domain instead of www.ifixit.com to avoid going through cloudfront.

## QA
Make sure the product page still loads normally. Verify that revalidation time is generally faster than it was before. Verify that we still get cache hits after revalidation. (Note: product page redis cache entries become stale after 1 minute)

CC @sterlinghirsh @dhmacs
